packages/docusaurus-preset/src/index.ts

```diff
@@ -29,11 +30,15 @@ export default function protosaurusPreset(
 ): Preset {
   const presetOpts = options;
   const rehypePlugin = [protoMessageRehypePlugin, { siteDir: context.siteDir }];
+  const remarkPlugin = [remarkPluginAnnotations, { siteDir: context.siteDir }];
 
   if (typeof presetOpts?.docs === 'object') {
     presetOpts.docs.rehypePlugins = presetOpts?.docs.rehypePlugins
       ? [...presetOpts?.docs.rehypePlugins, rehypePlugin]
       : ([rehypePlugin] as any);
+    presetOpts.docs.remarkPlugins = presetOpts?.docs.remarkPlugins
```

Comment:
A question, when we have another remark plugins, can we add it from docusaurus config? I mean this preset can have args?

Reply:
NVM seems like so. This is cool. Thanks!

Reply:
Yes, if we want to pass another options to override our default config, perhaps we can pass it from `docusaurus.config.js`, something like:

```js
{
  docs: {}, // docusaurus stuff
  protosaurus: {} // our stuff
}
```